quickwit-metastore/migrations/postgresql/1_create-indexes.up.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE indexes (
```

Comment:
My guess is that we are ok this migration will not upgrade a previously running version of Quickwit. Users might need to manually upgrade or create a new db and then export/import the metastore data.